tooling/support/android/src/test/kotlin/com/squareup/tooling/support/android/VariantExtractorsTest.kt

```diff
@@ -53,7 +53,11 @@ class VariantExtractorsTest {
       "All source paths must be relative to the project dir"
     )
     assertEquals(24, srcs.size, "Sources were missing")
-    assertTrue("No dependencies should be listed") { deps.isEmpty() }
+    // Filter out the kotlin-stdlib-jdk8 dependency
+    val filteredDeps = deps.filterNot {
+      it.target.contains("kotlin-stdlib")
```

Comment:
This is coming in because of the kotlin plugin adding in this library to the project during the configuration resolution.

Reply:
Yes so I had to exclude that from tests.